definitions/npm/directory-tree_v2.x.x/flow_v0.71.x-/directory-tree_v2.x.x.js

```diff
@@ -0,0 +1,30 @@
+import type pathType from 'path';
```

Comment:
put this inside module declaration, otherwise it sets it to `any`. See contributing guide for more info.

Reply:
@AndrewSouthpaw OK!